ci/jenkins/test-vmc.sh

```diff
@@ -629,6 +817,51 @@ function cleanup_cluster() {
     release_static_ip
 }
 
+function cleanup_aws_cluster() {
+    echo "=== Cleaning up AWS cluster ${CLUSTER} ==="
+    export KUBECONFIG=$KUBECONFIG_PATH
+
+    echo "=== Cleaning up Resources ===="
+    export AWS_CONTROLLER_IAM_ROLE=$AWS_SERVICE_USER_ROLE_ARN
+    clusterctl delete --infrastructure aws
+    export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile --region $AWS_REGION)
+    clusterctl init --infrastructure aws
+    
+    kubectl delete cluster ${CLUSTER} -n ${CLUSTER}
```

Comment:
After deleting the cluster, can it traverse though all kinds of resources in the cluster template, and `kubectl delete --all $KIND -n ${CLUSTER}`, and finally delete the namespace? If this still cannot delete the ec2 and LB instance, it can try deleting via ec2 and elb commands.